tests/test_init.py

```diff
@@ -19,90 +19,241 @@
-                      (qml.init.cvqnn_layers_phi_uniform, rpt_uni, (repeat, n_if)),
-                      (qml.init.cvqnn_layers_varphi_uniform, rpt_uni, (repeat, n_wires)),
-                      (qml.init.cvqnn_layers_r_uniform, rpt_uni, (repeat, n_wires)),
-                      (qml.init.cvqnn_layers_phi_r_uniform, rpt_uni, (repeat, n_wires)),
-                      (qml.init.cvqnn_layers_a_uniform, rpt_uni, (repeat, n_wires)),
-                      (qml.init.cvqnn_layers_phi_a_uniform, rpt_uni, (repeat, n_wires)),
-                      (qml.init.cvqnn_layers_kappa_uniform, rpt_uni, (repeat, n_wires)),
-                      (qml.init.interferometer_theta_uniform, base_uni, (n_if,)),
-                      (qml.init.interferometer_phi_uniform, base_uni, (n_if,)),
-                      (qml.init.interferometer_varphi_uniform, base_uni, (n_wires,)),
-                      (qml.init.qaoa_embedding_normal, rpt_nrml, (repeat, 2*n_wires)),
-                      (qml.init.qaoa_embedding_uniform, rpt_uni, (repeat, 2*n_wires)),
-                      (qml.init.qaoa_embedding_uniform, {'n_layers': 2, 'n_wires': 1, 'low': 0, 'high': 1}, (2, 1)),
-                      (qml.init.qaoa_embedding_uniform, {'n_layers': 2, 'n_wires': 2, 'low': 0, 'high': 1}, (2, 3)),
-                      (qml.init.qaoa_embedding_normal, {'n_layers': 2, 'n_wires': 1, 'mean': 0, 'std': 1}, (2, 1)),
-                      (qml.init.qaoa_embedding_normal, {'n_layers': 2, 'n_wires': 2, 'mean': 0, 'std': 1}, (2, 3)),
-                      (qml.init.simplified_two_design_initial_layer_uniform, {'n_wires': 1, 'low': 0, 'high': 1}, (1,)),
-                      (qml.init.simplified_two_design_initial_layer_uniform, {'n_wires': 3, 'low': 0, 'high': 1}, (3,)),
-                      (qml.init.simplified_two_design_initial_layer_normal, {'n_wires': 1, 'mean': 0, 'std': 1}, (1,)),
-                      (qml.init.simplified_two_design_initial_layer_normal, {'n_wires': 3, 'mean': 0, 'std': 1}, (3,)),
-                      (qml.init.simplified_two_design_weights_uniform, {'n_layers': 2, 'n_wires': 1, 'low': 0, 'high': 1}, (0,)),
-                      (qml.init.simplified_two_design_weights_uniform, {'n_layers': 2, 'n_wires': 2, 'low': 0, 'high': 1}, (2, 1, 2)),
-                      (qml.init.simplified_two_design_weights_uniform, {'n_layers': 2, 'n_wires': 4, 'low': 0, 'high': 1}, (2, 3, 2)),
-                      (qml.init.simplified_two_design_weights_normal, {'n_layers': 2, 'n_wires': 1, 'mean': 0, 'std': 1}, (0, )),
-                      (qml.init.simplified_two_design_weights_normal, {'n_layers': 2, 'n_wires': 2, 'mean': 0, 'std': 1}, (2, 1, 2)),
-                      (qml.init.simplified_two_design_weights_normal, {'n_layers': 2, 'n_wires': 4, 'mean': 0, 'std': 1}, (2, 3, 2)),
-                      (qml.init.basic_entangler_layers_normal, {'n_layers': 2, 'n_wires': 1, 'mean': 0, 'std': 1},
+INIT_KWARGS_SHAPES = [(qml.init.random_layers_normal,
```

Comment:
(Mino suggestion): placing small comments in the list could help out with readability. If this would become irrelevant with the changes that are planned, then this is not crucial :)
```suggestion
INIT_KWARGS_SHAPES = [
                       # Random layers parameters
                       (qml.init.random_layers_normal,
```